app/views/admin/feedback_messages/_latest_articles.html.erb

```diff
@@ -15,10 +15,10 @@
 
             <div class="ml-auto shrink-0">
               <a href="<%= article.path %>/mod" class="btn btn-secondary btn-sm">Mod</a>
-              <a href="<%= article.path %>/moderate" class="btn btn-secondary btn-sm">admin</a>
+              <a href="<%= article.path %>/moderate" class="btn btn-secondary btn-sm">Admin</a>
               <a href="<%= article.user.path %>" class="btn btn-secondary btn-sm">Profile</a>
-              <%= link_to "admin User", admin_user_path(article.user_id), class: "btn btn-secondary btn-sm" %>
-              <%= link_to "Destructive Actions", edit_admin_user_path(article.user_id), class: "btn btn-danger btn-sm" %>
+              <%= link_to "Admin User", admin_user_path(article.user_id), class: "btn btn-secondary btn-sm" %>
+              <%= link_to "Destructive Actions", admin_user_path(article.user_id), class: "btn btn-danger btn-sm" %>
```

Comment:
I had to change this link in here so while I was in the file, I updated "Admin" to be capitalized per conversations that I've had with product-focused folks on the team. Same goes for [here](https://github.com/forem/forem/pull/16764/files#diff-23851e83e6da68e48eb56f5b091e848080acd6f7d4991ed6df60fd4937e08552R15). 😊